cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -51,14 +51,14 @@ func NewKubeadmCommand(f *cmdutil.Factory, in io.Reader, out, err io.Writer, env
 			    ┌──────────────────────────────────────────────────────────┐
 			    │  On the first machine                                    │
 			    ├──────────────────────────────────────────────────────────┤
-			    │ master# kubeadm init master                              │
+			    │ master# kubeadm init                                     │
 			    │ Your token is: <token>                                   │
```

Comment:
Yes, I'll remove the output lines as it may become hard to manage in the future.